### PR TITLE
adding reliability metrics page

### PR DESCRIPTION
**Pull request by choldgraf**
I took a stab at making a "reliability / metrics" page. It still needs some content, but the basic structure is there. Here's a live version:

http://predictablynoisy.com/binder/reliability.html

It basically does two things:

1. Give some indication of what users should expect for `mybinder.org`. I haven't put anything concrete here yet since we should decide that collectively.
2. Embed two of the grafana plots that I made for the SLO dashboard.

Hopefully this is a start...would love to hear what people think!

### CODE REVIEW

File: doc/reliability.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+``mybinder.org`` status and reliability
+=======================================
+
+This page serves as a source of information for general Binder status
```
Comment (betatim) [inline doc/reliability.rst]: Should we try and encourage people to think of mybinder.org as something different from the Binder project and binderhub the software project? I'd change "general Binder status" to "mybinder.org status"

File: doc/reliability.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Site Reliability Goals
+----------------------
+
+``mybinder.org`` is a free service running on open-source software. It should
+be treated as "beta" technology. As such, the Binder team chooses reliability
+goals that are generally less than the gold-standard in industry. Below are
+some general reliability goals we aim for.
+
+* GOAL 1
+* GOAL 2
```
Comment (betatim) [inline doc/reliability.rst]: Can we change this to "We are still working on defining what the exact goals should be. Currently we label it as 'best effort'"? I think ahveing a simple page that shows a condensed view of the grafana dashboards that we could even use as target for status.mybinder.org is so useful that we should get it online before we workout goals that we are happy with.

**Comment by betatim** [comment]
I'd be happy to merge this after editing the goals section in the way I suggested to get a simple status page up and running soon.

**Comment by choldgraf** [comment]
I'm +1 on this @betatim , though let's see if @willingc or @yuvipanda are OK with that, since they've been in the "reliability goals" conversation too :-)

**Comment by choldgraf** [comment]
ok @betatim - merged your suggestion in. I agree we can figure this out over time

File: doc/reliability.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
+``mybinder.org`` status and reliability
+=======================================
+
+This page serves as a source of information for general Binder status
+updates, building/launching statistics, and reliability metrics.
+
+Site Reliability Goals
```
Comment (willingc) [inline doc/reliability.rst]: I would add a section: "Expectations for mybinder.org users"
Let's use some of Fernando's wording

---
*ABSOLUTELY*  This is, for now, purely a best-effort, experimental research project. We're *not* staffed 24/7, nor do we have the budget, capacity or intent at this point to go there.  

Communicating this clearly on the homepage could help: to say that this is a *research pilot*, whose main goal is to understand usage patterns and workloads for future evolution, but *not* a service that anyone can rely on for anything critical, would go a long way.

File: doc/reliability.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+----------------------
+
+``mybinder.org`` is a free service running on open-source software. It should
```
Comment (willingc) [inline doc/reliability.rst]: As mybinder.org is a research pilot project, the main goal for the project is to understand usage patterns and workloads for future project evolution. While we strive for site reliability and availability, we want our users to understand the intent of this service is research and we offer no guarantees of its performance in mission critical uses.

File: doc/reliability.rst (hunk 3 above)
Comment (yuvipanda) [reply doc/reliability.rst]: Where is this text from? :)

**Comment by choldgraf** [comment]
@willingc those are great suggestions, incorporated into latest push...I also added a note on the main page, LMKWYT

File: doc/reliability.rst (hunk 3 above)
Comment (willingc) [reply doc/reliability.rst]: From @fperez's wonderful mind @yuvipanda 👍

**Comment by willingc** [comment]
Thanks @choldgraf! LGTM

**Comment by betatim** [comment]
🔮  (because @fperez mind reading)

**Pull request merged**